src/input/validators.ts

```diff
@@ -14,3 +14,13 @@ export const choiceValidator = (
   numberOfChoices: number,
   errorMessage: string
 ) => (value > 0 && value <= numberOfChoices) || errorMessage
+
+// This regex will match special missing
+// `a-z` or `_` or `.` or '.a'
+export const isSpecialMissing = (value: any) => {
+  // This regex can't cover the case with `..` two dots, so if we receive it that is false special missing
+  if (typeof value === 'string' && value.includes('..')) return false
+
+  const regex = new RegExp('^(\\.)?[a-z_.]{1}$', 'i')
```

Comment:
this reg exp doesn't match `.a`

Reply:
![image](https://user-images.githubusercontent.com/18329105/163809949-97df69b2-f115-4a0a-a0cc-dc9dce0ca5ae.png)

Keep in mind the double backslash `\\` because we escape it in JS. If you copy it in some online regex tester, one backslash should be removed.

Reply:
yes, you are right.